usaspending_api/broker/helpers.py

```diff
@@ -322,20 +329,61 @@ def get_business_categories(row, data_type):
             business_category_set.add('authorities_and_commissions')
 
         if business_category_set & {'regional_and_state_government', 'us_territory_or_possession', 'local_government',
-                                    'indian_native_american_tribal_government', 'authorities_and_commissions'}:
+                                    'indian_native_american_tribal_government', 'authorities_and_commissions',
+                                    'regional_organization'}:
             business_category_set.add('government')
 
+        # INDIVIDUALS
         if business_types in ('P', '21'):
             business_category_set.add('individuals')
 
         return list(business_category_set)
 
     elif data_type == 'fpds':
         legal_entity_bool_dict = build_legal_entity_booleans_dict(row)
-        # SMALL BUSINESS
-        if legal_entity_bool_dict['small_business_competitive'] is True \
-                or legal_entity_bool_dict['for_profit_organization'] is True:
-            business_category_set |= {'small_business', 'category_business'}
+
+        # BUSINESS (FOR-PROFIT)
+        if row.get('contracting_officers_deter') == 'S' \
+                or legal_entity_bool_dict['women_owned_small_business'] is True \
+                or legal_entity_bool_dict['economically_disadvantaged'] is True \
+                or legal_entity_bool_dict['joint_venture_women_owned'] is True \
+                or legal_entity_bool_dict['emerging_small_business'] is True \
+                or legal_entity_bool_dict['self_certified_small_disad'] is True \
+                or legal_entity_bool_dict['small_agricultural_coopera'] is True \
+                or legal_entity_bool_dict['small_disadvantaged_busine'] is True:
+            business_category_set |= {'small_business'}
```

Comment:
Why are the types sometimes added via `.add()` and other times using the union operator `|=`?

Reply:
Good call, never noticed that. I'll stick with the consistent `.add()`